### PR TITLE
Mesh interaction

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
Mesh Interaction and optimizing spaces (I'm sorry)

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
This commit will fix pixi-spine issue https://github.com/pixijs/pixi-spine/issues/38

**Comment by englercj** [comment]
:+1: Looks good to me, @GoodBoyDigital thoughts?

File: src/mesh/Mesh.js [hunk 1]
```diff
@@ -366,55 +368,103 @@ Mesh.prototype._onTextureUpdate = function ()
...
+ * Tests if a point is inside this mesh. Works only for TRIANGLE_MESH
+ *
+ * @param point {Point} the point to test
+ * @return {boolean} the result of the test
+ */
+Mesh.prototype.containsPoint = function( point ) {
+    if (!this.getBounds().contains(point.x, point.y)) {
+        return false;
+    }
+    this.worldTransform.applyInverse(point,  tempPoint);
```
Comment (endel) [inline src/mesh/Mesh.js]: Two spaces on function arguments

**Comment by endel** [comment]
Wouldn't it be nice to have some tests on this? 

It may have some assertions that `containsPoint` does what it is supposed to do. It seems implemented perfectly, but having tests it will ensure that future changes doesn't break existing functionality.

**Comment by englercj** [comment]
I 100% agree, I don't think it should stop the merge of the feature, but we should definitely add some if at all possible.

**Comment by ivanpopelyshev** [comment]
Ok, I'll make some tests :) Btw, I only did test with vertex indices, other branch of "if" wasnt tested.

**Comment by GoodBoyDigital** [comment]
looks good to me! happy to roll in!

**Pull request merged**